README.md

```diff
@@ -193,6 +193,23 @@ $ $EDITOR ~/.dcrd/dcrd.conf
 $ $EDITOR ~/.dcrwallet/dcrwallet.conf
 ```
 
+### Docker
+
+You can also run the wallet with docker:
+
+Volumes:
+
+- `/etc/dcrwallet` **mandatory** hold `config`, `rpc.cert ` and `rpc.key`
+- `/var/log/dcrwallet` is where the log files are stored
+- `/var/lib/dcrwallet` **important** hold the private keys, don't keep that volume ogn a temporary file system.
+
+Ports:
+
+- `9110` gRPC
+- `9111` Legacy RPC
```

Comment:
these are reversed, 9110/19110 is the json-rpc (legacy) port

Reply:
fixed https://github.com/decred/dcrwallet/pull/999/commits/a0bf967daf7cdff5fa492f8e04b5e5f400180aed